db/seeds.rb

```diff
@@ -0,0 +1,22 @@
+User.create!([
+  {email: "bob@test.com", encrypted_password: "$2a$10$0yf0n7XnZDo5e/lMIzQVM.ulET9UGL9c/V.5fuYS8uwEHe9jvEuaO", reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 1, current_sign_in_at: "2015-09-07 19:42:45", last_sign_in_at: "2015-09-07 19:42:45", current_sign_in_ip: "::1", last_sign_in_ip: "::1"},
+  {email: "ben@test.com", encrypted_password: "$2a$10$PnBtoDRQALhAUPgYYopH0OqNeLO/r4LOOTHkL86qqquf0MA958a.C", reset_password_token: nil, reset_password_sent_at: nil, remember_created_at: nil, sign_in_count: 3, current_sign_in_at: "2015-09-10 13:21:37", last_sign_in_at: "2015-09-08 07:10:49", current_sign_in_ip: "::1", last_sign_in_ip: "::1"}
```

Comment:
Line is too long. [343/80]<br>Space inside { missing.<br>Space inside } missing.